walkthroughs/week-3-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBNSx0u-nLQKS8kHvzdNjHeCTBKmQONoP4"></script>
```

Comment:
Why do we need to make changes in the walkthroughs folder?

Reply:
Actually, in the walkthrough they made us do this change. But will revert it as it is not needed.

Reply:
Thank you for the clarification!